sdks/wasm/binding_support.js

```diff
@@ -6,7 +6,8 @@ var BindingSupportLib = {
 		mono_wasm_object_registry: [],
 		mono_wasm_ref_counter: 0,
 		mono_wasm_free_list: [],
-		mono_wasm_marshal_enum_as_int: false,	
+		mono_wasm_marshal_enum_as_int: false,
+		mono_wasm_core_map: {},	
```

Comment:
If this is a Map (or WeakMap) normally, initializing it to {} is wrong and will break. Might as well init it to null.

Reply:
Addressed in recent commit